edges.go

```diff
@@ -74,7 +76,7 @@ func (this Edges) Sorted() []*Edge {
 		}
 		sort.Strings(dsts)
 		for _, dst := range dsts {
-			edges = append(edges, this.SrcToDsts[src][dst])
+			edges = append(edges, this.SrcToDsts[src][dst]...)
```

Comment:
We should probably take into account the Attrs when sorting, since each src and dst can now have multiple edges.